src/psyclone/transformations.py

```diff
@@ -596,31 +596,31 @@ class OMPTargetTrans(RegionTrans):
             integer, dimension(10, 10) :: A
             integer :: i
             integer :: j
-            $!omp target
+            !$omp target
             do i = 1, 10
                 do j = 1, 10
                     A(i, j) = 0
                 end do
             end do
-            $!omp end target
+            !$omp end target
         end subroutine
 
     '''
-    def apply(self, nodes, options=None):
-        ''' Insert a OMPTargetDirective before the provided node or list
+    def apply(self, node, options=None):
+        ''' Insert an OMPTargetDirective before the provided node or list
         of nodes.
 
-        :param nodes: the PSyIR node or nodes to enclose in the OpenMP \
+        :param node: the PSyIR node or nodes to enclose in the OpenMP \
                       target region.
-        :type nodes: list of :py:class:`psyclone.psyir.nodes.Node`
+        :type node: list of :py:class:`psyclone.psyir.nodes.Node`
```

Comment:
Perhaps "(list of) ...." as it might not be a list.

Reply:
Done